server/db/data/recipedia.schema

```diff
@@ -56,4 +72,7 @@ vitamin_b6: float .
 magnesium: float .
 folate: float .
 
+cname: string .
+
 contains: [uid] @reverse .
+categorized_as: uid .
```

Comment:
This is supposed to be a list of categories?

Reply:
I will keep the name of the relationship and make it a list so it's more extendable in the future.